src/components/composer/formula_assistant/formula_assistant.xml

```diff
@@ -1,6 +1,12 @@
 <templates>
   <t t-name="o-spreadsheet-FunctionDescriptionProvider">
-    <div class="o-formula-assistant-container user-select-none shadow">
+    <div
+      class="o-formula-assistant-container shadow"
+      tabindex="0"
```

Comment:
Also pressing tab when the assistant is focused:
- on the grid composer puts the focus on the grid scrollbar(?), then back on the composer
- on the topbar composer it puts the focus on the "add rows footer", breaking the display
<img width="512" height="203" alt="image" src="https://github.com/user-attachments/assets/2e9a5c45-e366-4810-9802-2ffa4ff97944" />

Reply:
I guess blocking every keyboard input would do the trick :p I guess we would only miss the "Ctrl.A" feature ( and I really don't see the point of doing this tbh)

Reply:
the issue I see with the onBlus is that is some real workd cases, like switching tabs to copy/paste a content, you automatically lose the focus, which closes the composer. I stumbled upon it a few times and it wasa bit frustrating. And I obviously won't mention the debug behaviour.  I will work around it for this task ofc

Reply:
I have no strong feeling on onBlur in the grid/topbar composer either way, if you can convince francois i'm fine with removing it :shrug: